File: PHP/car.php
<?php

require_once ('account.php');
require_once ('car.php');

class Car{
    public $id;
    public $license;
    public $driver;
    public $passanger;

    public function __construct($license, $driver)
    {
        $this->license = $license;
        $this->driver = $driver;
    }

    public function printDataCar() {
        echo " | License: " . $this->license; 
        echo " | Driver: " . $this->driver->name;
        echo " | Document: " . $this->driver->document;
        
    }

}

?>

File: Java/Car.java
package Java;

class Car {
    private Integer id;
    private String license;
    private Account driver;
    private Integer passengers;

    public Car(String license, Account driver){
        this.license = license;
        this.driver = driver;
    }

    void printDataCar(){
        if(passengers != null){
            System.out.println("License: " + license + " Name driver: " + driver.name + " Passengers: " + passengers);
        }
    }

    public Integer getPassengers(){
        return passengers;
    }

    public void setPassengers(Integer passengers){
        if(passengers == 4){
            this.passengers = passengers;
        }else{
            System.out.println("Only 4 passengers");
        }
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getLicense() {
        return license;
    }

    public void setLicense(String license) {
        this.license = license;
    }

    public Account getDriver() {
        return driver;
    }

    public void setDriver(Account driver) {
        this.driver = driver;
    }

    
}
